Civi/Api4/Contact.php

```diff
@@ -93,13 +93,34 @@ public static function getDuplicates($checkPermissions = TRUE) {
 
   /**
    * @param bool $checkPermissions
+   *
    * @return Action\Contact\MergeDuplicates
    */
-  public static function mergeDuplicates($checkPermissions = TRUE) {
+  public static function mergeDuplicates(bool $checkPermissions = TRUE): Action\Contact\MergeDuplicates {
     return (new Action\Contact\MergeDuplicates(self::getEntityName(), __FUNCTION__))
       ->setCheckPermissions($checkPermissions);
   }
 
+  /**
+   * @param bool $checkPermissions
+   *
+   * @return Action\Contact\getMergedTo
+   */
+  public static function getMergedTo(bool $checkPermissions = TRUE): Action\Contact\getMergedTo {
+    return (new Action\Contact\getMergedTo(self::getEntityName(), __FUNCTION__))
+      ->setCheckPermissions($checkPermissions);
+  }
+
+  /**
+   * @param bool $checkPermissions
+   *
+   * @return Action\Contact\getMergedFrom
+   */
+  public static function getMergedFrom(bool $checkPermissions = TRUE): Action\Contact\getMergedFrom {
```

Comment:
Pretty sure all 4 `g`s should be capitalized here.

Reply:
@colemanw Hmm - capitalizing 3 makes sense but the main fn should be `  public static function getMergedFrom(bool $checkPermissions = TRUE): Action\Contact\getMergedFrom {` I think....?

Reply:
I would have thought it should be `public static function getMergedFrom(bool $checkPermissions = TRUE): Action\Contact\GetMergedFrom` given that the class name is a capital first letter here https://github.com/civicrm/civicrm-core/blob/master/Civi/Api4/Action/Contact/GetMergedFrom.php

Reply:
@seamuslee001 I think it's right now